.travis.yml

```diff
@@ -1,23 +1,17 @@
 ---
 language: node_js
 node_js:
-  - "0.12"
+  - "4"
 
 sudo: false
 
-addons:
-  apt:
-    sources:
-    - google-chrome
-    packages:
-    - google-chrome-stable
```

Comment:
I considered it, do you know if this was tested in chrome for any particular reason. The tests are super-simple, and it seemed a waste of time to download and install chrome in travis for some tests with no UI.

But I surely can, just double checking.